docs/src/autotuning-ridge.md

```diff
@@ -0,0 +1,211 @@
+```@example 3
+using DiffOpt
+using Statistics
+using OSQP
+using JuMP
+using Plots
+import Random
+using LinearAlgebra
+using Printf
+Base.show(io::IO, f::Float64) = @printf(io, "%1.3f", f) # to reduce float precision while printing
```

Comment:
We should not add this line and find another way where necessary

Reply:
ok. removed that line. anyway i used it only while debugging